.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```

Comment:
What would this be?

Reply:
This is just the default setting. So when you create a new issue with this template, the default title will be blank. This setting lets you specify the default, perhaps a prefix like the BUG one.

Reply:
Oh, does this interact with the GitHub API somehow?

Reply:
I answered my own question and read the docs for issue templates. Never mind, I get it now. 😆